### DISCUSSION TITLE
Zoom Speed

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#4257] [Q&A] [answered]
Greetings,
Is there any way to decrease the speed at which the Viewer is zoomed in and out using mouse scroll or gestures?

**Comment by pmconne** [comment, marked as answer]
Try reducing [ToolSettings.wheelZoomRatio](https://www.itwinjs.org/reference/core-frontend/tools/toolsettings/). Note it has a minimum of 1.0; default is 1.5.

**Comment by bbastings** [comment]
If you are in a perspective view, make sure the mouse cursor is over some geometry (element, terrain, etc.) and not empty space as this provides the depth plane to zoom about. For pinch zoom. the geometry half-way between your fingers is used.

I mention this in case the issue isn't zoom speed...
